docs/dev/zksync-v2/blocks-and-time.md

```diff
@@ -1,3 +1,24 @@
 # Block numbers and time
 
-TODO.
+On ZkSync there are two types of blocks: L2 blocks and L1 batches.
+
+## L2 blocks
+
+L2 block is similar to a common block on L1. L2 block time is almost constant, like a few seconds.
+In this documentation by default the word "block" actually means "L2 block" including API.
+L2 block hash is deterministic and is basically equal to `keccak256(bytes4(l2_block_number))`.
```

Comment:
Maybe let's have it as `keccak256(l2_block_number)`?

IMHO does not really matter, but the `bytes4` part complicates the thing a bit. Also, formally, there is a difference between 

```
keccak256(bytes.concat(bytes4(l2_block_number)))
```

and 

```
keccak256(bytes.concat(bytes32(uint256(l2_block_number))))
```

The former is left-aligned, while the latter is right-aligned. We don't need to elaborate that much in the documentation, but I am still curious :)